notebooks/Google-Colab-PFLD-Grove-Example.ipynb

```diff
@@ -51,34 +48,32 @@
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        },
-        "id": "F5pMIwXuvPpG",
-        "outputId": "d66ee6b7-6398-4d0a-a78b-4cf294a4b4e2"
+        "id": "F5pMIwXuvPpG"
       },
       "outputs": [],
       "source": [
+        "# Ethos-U-Vela need to be installed this way, or SSCMA does not work anymore...\n",
+        "!git clone https://review.mlplatform.org/ml/ethos-u/ethos-u-vela.git\n",
```

Comment:
Could we just install the stable release from [https://pypi.org/project/ethos-u-vela](https://pypi.org/project/ethos-u-vela/)?